config/const.py

```diff
@@ -1,13 +1,17 @@
 from os import sep
 from os.path import join, dirname, realpath
```

Comment:
Use pathlib instead.

Reply:
Using pathlib would have resulted in code flaws, so I left it as a Join, which is a String object.